src/views/Miscellaneous.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-menu-button slot="start" />
+        <ion-title>{{ $t("Miscellaneous") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <main>
+        <section>
+          <ion-list>
+            <ion-list-header>{{ $t("Miscellaneous jobs") }}</ion-list-header>
+            <ion-item v-for="job in miscellaneousJobs" :key="job.jobId" @click="viewJobConfiguration(job)">
+              <ion-label>{{ job.jobName }}</ion-label>
+              <ion-badge v-if="job.runTime" color="light">{{ timeFromNow(job.runTime)}}</ion-badge>
+              <ion-icon slot="end" :icon="chevronForwardOutline" />
+            </ion-item>
+          </ion-list>
+        </section>
+
+        <aside class="desktop-only" v-if="isDesktop" v-show="currentJob && Object.keys(currentJob).length">
+          <JobConfiguration :title="title" :status="currentJobStatus" :type="freqType" :key="currentJob"/>
+        </aside>
+      </main>
+    </ion-content>
+  </ion-page>      
+</template>
+<script lang="ts">
+import { DateTime } from 'luxon';
+import {
+  IonBadge,
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonMenuButton,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  isPlatform,
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { useStore } from "@/store";
+import { useRouter } from 'vue-router'
+import { mapGetters } from "vuex";
+import emitter from '@/event-bus';
+import JobConfiguration from '@/components/JobConfiguration.vue';
+import { chevronForwardOutline } from "ionicons/icons";
+
+export default defineComponent({
+  name: 'Miscellaneous',
+  components: {
+    IonBadge,
+    IonContent,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonMenuButton,
+    IonPage,
+    IonTitle,
+    IonToolbar,
+    JobConfiguration
+  },
+  mounted() {
+    this.getMiscellaneousJobs();
+  },
+  data() {
+    return {
+      jobEnums: JSON.parse(process.env?.VUE_APP_ODR_JOB_ENUMS as string) as any,
+      batchJobEnums: JSON.parse(process.env?.VUE_APP_BATCH_JOB_ENUMS as string) as any,
+      jobFrequencyType: JSON.parse(process.env?.VUE_APP_JOB_FREQUENCY_TYPE as string) as any,
+      webhookEnums: JSON.parse(process.env?.VUE_APP_WEBHOOK_ENUMS as string) as any,
```

Comment:
Remove this property, as it's not needed on this page, also check for other properties as well.

Reply:
Sure sir.